common/network-common/src/main/java/org/apache/spark/network/client/TransportClientFactory.java

```diff
@@ -112,6 +114,7 @@ public TransportClientFactory(
     }
     this.metrics = new NettyMemoryMetrics(
       this.pooledAllocator, conf.getModuleName() + "-client", conf);
+    fastFailTimeWindow = conf.ioRetryWaitTimeMs() * 0.95;
```

Comment:
I just have another idea to make code a bit more simpler:

```suggestion
    fastFailTimeWindow = if (conf.maxIORetries() > 0) conf.ioRetryWaitTimeMs() * 0.95 else 0;
```

Then, we can simply the if condition below and call `conf.maxIORetries()` only once.

Reply:
thanks